frontend/src/app/modules/time-series/services/linechart-data.service.ts

```diff
@@ -0,0 +1,67 @@
+import {Injectable} from '@angular/core';
+import {HttpClient, HttpParams} from "@angular/common/http";
+import {ResultSelectionCommand} from "../../result-selection/models/result-selection-command.model";
+import {RemainingResultSelection} from "../../result-selection/models/remaing-result-selection.model";
+import {EMPTY, Observable, OperatorFunction} from "rxjs";
+import {GetBarchartCommand} from "../../aggregation/models/get-barchart-command.model";
+import {catchError} from "rxjs/operators";
+import {GetLinechartCommand} from "../models/get-line-chart-command.model";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LinechartDataService {
+
+  constructor(private http: HttpClient) { }
+
+  fetchLinechartData<T>(resultSelectionCommand: ResultSelectionCommand, remainingResultSelection: RemainingResultSelection, url: string): Observable<T> {
+    const getLinechartCommand = LinechartDataService.buildGetLinechartCommand(resultSelectionCommand, remainingResultSelection);
+    const params = this.createParams(getLinechartCommand);
+    return this.http.get<T>(url, {params: params}).pipe(
+      this.handleError()
+    )
+  }
+
+  private static buildGetLinechartCommand(resultSelectionCommand: ResultSelectionCommand, remainingResultSelection: RemainingResultSelection): GetLinechartCommand {
```

Comment:
This method should be converted into an instance method I think. Its signature would become a bit simpler because its parameters are accessible as fields of the service instance. 
The only call of the method is from the _fetchLinechartData_ (instance) method of the service, actually. Until there are other calls which would benefit from a static method without the need to avoid DI I can't see why a static method should make sence.